.github/workflows/build.yml

```diff
@@ -83,6 +83,7 @@ jobs:
             --install defcon \
             --install gadopt \
             --install asQ \
+            --package-branch pyadjoint dolci/disk_checkpointing \
```

Comment:
The test passed with `--package-branch pyadjoint dolci/disk_checkpointing`. I will remove it to merge this PR.